packages/forklift-console-plugin/src/modules/Plans/modals/PlanCutoverMigrationModal.tsx

```diff
@@ -160,16 +158,25 @@ export const PlanCutoverMigrationModal: React.FC<PlanCutoverMigrationModalProps>
       actions={actions}
     >
       <>
-        <ForkliftTrans>
-          <p>
-            Schedule the cutover for migration <strong className="co-break-word">{name}</strong>?
-          </p>
-          <br />
-          <p>
-            You can schedule cutover for now or a future date and time. VMs included in the
-            migration plan will be shut down when cutover starts.
-          </p>
-        </ForkliftTrans>
+        {cutoverSet ? (
+          <ForkliftTrans>
+            <p>
+              Edit the cutover for migration <strong className="co-break-word">{name}</strong>?
+            </p>
+          </ForkliftTrans>
+        ) : (
+          <ForkliftTrans>
+            <p>
+              Schedule the cutover for migration <strong className="co-break-word">{name}</strong>?
+            </p>
+          </ForkliftTrans>
+        )}
```

Comment:
This can be simply:
```tsx
<ForkliftTrans><p>{cutoverSet ? "Edit" : "Schedule"} the cutover for migration <strong className="co-break-word">{name}</strong>? </p></ForkliftTrans>
```